NEWS.md

```diff
@@ -17,6 +17,8 @@ character vector (`'--DTC'`), was imputed. (#2146)
 were enhanced such that more than one summary variable can be derived, e.g.,
 `AVAL` as the sum and `ADT` as the maximum of the contributing records. (#1792)
 
+- `derive_summary_records()` was enhanced with the following optional arguments: `dataset_add`, `dataset_ref`, `missing_values`. These arguments respectively, generate summary variables from additional datasets, retain/add specific records from a reference dataset, and impute user-defined missing values. (#2142)
+
```

Comment:
We should add that
- `dataset` is optional now (in `derive_summary_records()` and `derive_param_exposure()`)
- `dataset_add` was added to `derive_param_exposure()`